foundation_cms/templates/patterns/components/footer.html

```diff
@@ -34,31 +34,38 @@
3984C24.7415 16.1175 24.7547 15.7086 24.7485 12.4504C24.7422 9.19211 24.7275 8.78361 24.6668 7.50391C24.6061 6.22421 24.4028 5.35521 24.1035 4.59221C23.7951 3.80331 23.3835 3.13541 22.7159 2.46941C22.0482 1.80341 21.378 1.39421 20.5878 1.08991C19.824 0.794405 18.9517 0.593105 17.6744 0.537905C16.3971 0.482705 15.986 0.468405 12.727 0.474805C9.46798 0.481005 9.05998 0.495005 7.78008 0.557405ZM7.92028 22.2505C6.75028 22.1996 6.11498 22.0052 5.69158 21.8425C5.13098 21.6265 4.73158 21.3654 4.30968 20.9475C3.88768 20.5297 3.62858 20.1289 3.40968 19.5695C3.24528 19.1461 3.04728 18.5115 2.99258 17.3415C2.93308 16.077 2.92058 15.6973 2.91358 12.4935C2.90658 9.28981 2.91888 8.91051 2.97428 7.64551C3.02428 6.47651 3.21988 5.84051 3.38228 5.41731C3.59828 4.85601 3.85848 4.45731 4.27728 4.03571C4.69608 3.61401 5.09568 3.35431 5.65558 3.13541C6.07858 2.97031 6.71308 2.77401 7.88258 2.71831C9.14808 2.65831 9.52728 2.64631 12.7306 2.63931C15.9339 2.63231 16.3141 2.64431 17.5801 2.70011C18.7491 2.75091 19.3854 2.94461 19.8081 3.10811C20.3689 3.32411 20.7681 3.58351 21.1897 4.00311C21.6114 4.42251 21.8713 4.82071 22.0902 5.38181C22.2555 5.80351 22.4519 6.43781 22.5071 7.60811C22.5673 8.8736 22.581 9.25311 22.5867 12.4561C22.5925 15.6591 22.5812 16.0395 22.5257 17.3041C22.4747 18.4741 22.2807 19.1096 22.1177 19.5335C21.9017 20.0939 21.6414 20.4935 21.2223 20.9149C20.8033 21.3364 20.4042 21.596 19.844 21.8149C19.4216 21.9798 18.7863 22.1766 17.6178 22.2323C16.3522 22.2918 15.973 22.3043 12.7685 22.3113C9.56398 22.3183 9.18578 22.3053 7.92028 22.2505ZM17.703 6.05981C17.7035 6.34464 17.7884 6.62293 17.947 6.85948C18.1057 7.09604 18.3309 7.28023 18.5943 7.38876C18.8576 7.4973 19.1472 7.5253 19.4265 7.46922C19.7057 7.41314 19.9621 7.27551 20.1631 7.07372C20.3641 6.87194 20.5008 6.61507 20.5558 6.33561C20.6109 6.05615 20.5818 5.76664 20.4723 5.50371C20.3628 5.24077 20.1777 5.01623 19.9406 4.85846C19.7034 4.7007 19.4248 4.61681 19.14 4.61741C18.7582 4.6182 18.3923 4.77059 18.1228 5.04108C17.8534 5.31157 17.7023 5.678 17.703 6.05981ZM6.58848 12.4854C6.59518 15.8886 9.35908 18.6411 12.7615 18.6347C16.1641 18.6282 18.9185 15.8646 18.9121 12.4614C18.9056 9.05821 16.1411 6.30491 12.7381 6.31161C9.33508 6.31831 6.58208 9.08261 6.58848 12.4854ZM8.74998 12.4811C8.74842 11.69 8.98149 10.9162 9.41971 10.2575C9.85793 9.59885 10.4816 9.08493 11.2119 8.78073C11.9422 8.47654 12.7463 8.39573 13.5226 8.54853C14.2988 8.70133 15.0123 9.08088 15.5728 9.63917C16.1333 10.1975 16.5157 10.9094 16.6716 11.685C16.8274 12.4607 16.7498 13.2651 16.4485 13.9966C16.1472 14.7281 15.6358 15.3538 14.9789 15.7946C14.322 16.2355 13.5491 16.4716 12.758 16.4732C12.2327 16.4743 11.7123 16.3719 11.2266 16.1719C10.7408 15.9719 10.2992 15.6781 9.92706 15.3074C9.55487 14.9367 9.25935 14.4963 9.05739 14.0113C8.85542 13.5264 8.75097 13.0064 8.74998 12.4811Z" fill="white"/></g><defs><clipPath id="clip0_677_6983"><rect width="24" height="24" fill="white" transform="translate(0.75 0.473389)"/></clipPath></defs></svg>
```

Comment:
Are we animating the SVG itself here? If not, could we link to the file instead of inlining it?